corekit/src/main/java/com/midtrans/sdk/corekit/core/api/merchant/model/checkout/request/specific/creditcard/CreditCard.java

```diff
@@ -4,16 +4,26 @@
 
 import android.text.TextUtils;
 
+import com.midtrans.sdk.corekit.base.enums.Authentication;
+import com.midtrans.sdk.corekit.base.enums.BankType;
+import com.midtrans.sdk.corekit.base.enums.CreditCardType;
+import com.midtrans.sdk.corekit.utilities.Helper;
+
 import java.io.Serializable;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
+
+import static com.midtrans.sdk.corekit.utilities.Helper.mappingToBankType;
+import static com.midtrans.sdk.corekit.utilities.Helper.mappingToCreditCardAuthentication;
+import static com.midtrans.sdk.corekit.utilities.Helper.mappingToCreditCardType;
 
 public class CreditCard implements Serializable {
 
     public static final String MIGS = "migs";
```

Comment:
please create a new enum for acquiring channel that includes migs and no channel so if in the future we need to add another channel it's already available